sklearn/model_selection/_validation.py

```diff
@@ -257,19 +314,20 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
 
     else:
         fit_time = time.time() - start_time
-        test_score = _score(estimator, X_test, y_test, scorer)
+        test_scores = _score(estimator, X_test, y_test, scorers)
         score_time = time.time() - start_time - fit_time
         if return_train_score:
-            train_score = _score(estimator, X_train, y_train, scorer)
+            train_scores = _score(estimator, X_train, y_train, scorers)
 
     if verbose > 2:
-        msg += ", score=%f" % test_score
+        for scorer_name, score in test_scores.items():
+            msg += ", %s=%s" % (scorer_name, score)
     if verbose > 1:
         total_time = score_time + fit_time
         end_msg = "%s, total=%s" % (msg, logger.short_format_time(total_time))
         print("[CV] %s %s" % ((64 - len(end_msg)) * '.', end_msg))
 
-    ret = [train_score, test_score] if return_train_score else [test_score]
+    ret = [train_scores, test_scores] if return_train_score else [test_scores]
```

Comment:
can we return a dict instead and remove the "return stuff" parameters and just return everything all the time?